fs_brokers/apache_hdfs_broker/src/main/java/com/starrocks/broker/hdfs/FileSystemManager.java

```diff
@@ -450,6 +451,7 @@ public BrokerFileSystem getS3AFileSystem(String path, Map<String, String> proper
         String accessKey = properties.getOrDefault(FS_S3A_ACCESS_KEY, "");
         String secretKey = properties.getOrDefault(FS_S3A_SECRET_KEY, "");
         String endpoint = properties.getOrDefault(FS_S3A_ENDPOINT, "");
+        String pathStyleAccess = properties.getOrDefault(FS_S3A_PATH_STYLE_ACCESS, "true");
```

Comment:
Default needs to be "false" for backwards compatibility.

Reply:
done